diff --git a/node_modules/@equilibria/perennial-vaults/contracts/BalancedVault.sol b/node_modules/@equilibria/perennial-vaults/contracts/BalancedVault.sol
index 796f2b2..a1897f1 100644
--- a/node_modules/@equilibria/perennial-vaults/contracts/BalancedVault.sol
+++ b/node_modules/@equilibria/perennial-vaults/contracts/BalancedVault.sol
@@ -1,5 +1,5 @@
 //SPDX-License-Identifier: Apache-2.0
-pragma solidity 0.8.17;
+pragma solidity ^0.8.13;
 
 import "./interfaces/IBalancedVault.sol";
 import "@equilibria/root/control/unstructured/UInitializable.sol";
